pkg/reconciler/instances/eventing/preaction/removenatsoperatorstep.go

```diff
@@ -2,11 +2,13 @@ package preaction
 
 import (
 	"context"
+	"github.com/kyma-incubator/reconciler/pkg/reconciler/kubernetes/progress"
```

Comment:
(nit) group the similar imports together as in go standards.

Reply:
done